crates/forge/src/cheatcodes_hint_processor/file_operations.rs

```diff
@@ -37,10 +86,9 @@ fn string_into_felt(string: &str) -> Result<Felt252, ()> {
         let length = string.len();
         let first_char = string.chars().next();
         let last_char = string.chars().nth(length - 1);
-
         if length >= 2
             && length - 2 <= 31
-            && first_char == Some('\'')
+            && (first_char == Some('\'') || first_char == Some('\"'))
```

Comment:
What's the purpose of this check?

Reply:
the think is, serde_json returns String as String("sth"), in plain file reading we probably have String('sth') or String("sth") (@piotmag769 could you confirm?). For strings, we remove the first and last char before mapping to Felts. So here we make sure that we remove proper chars

Reply:
Felts can be written in 2 formats:
- integers
- shortstrings

If the parsed string can be parsed to integer, we check if it in is range [0, P).
If not, we check if it is a shortstring, which obviously starts with `'`. However, JSON grammar requires `"` for strings, therefore we allow both formats, since in txt files it is cleaner to just write using shortstrings using `'`e.g. `1 'hello' 3`

Reply:
Thanks